### ISSUE TITLE
The `set-env` command is deprecated and will be disabled on November 16th

### ISSUE THREAD

**Issue by JesperNJessen** [actions/setup-dotnet#144]
### Description

index.js uses set-env and add-path. Both will be disabled soon. As a result I'm seeing these warnings in my log:

```
The `set-env` command is deprecated and will be disabled on November 16th. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
The `add-path` command is deprecated and will be disabled on November 16th. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

### Details

YAML
```
name: Build on push

on:
  push:
    branches: [master]

jobs:

  Box:
    runs-on: <runner name>
    steps:
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0 
    - uses: actions/setup-dotnet@v1
      with:
        dotnet-version: '3.1.100'
    - uses: nuget/setup-nuget@v1
      with:
        nuget-version: 'latest'
    - name: <build name>
      run: |
        cd <Project name>
        nuget restore <Solution name>.sln
        msbuild  <Solution name>.sln /verbosity:normal /t:Rebuild /p:Configuration=Debug
        dotnet test  <Solution name>.sln

```

**Comment by r3econ** [comment]
They will be disabled in 5 days. Is this repository maintained?

**Comment by ericsampson** [comment]
According to the releases/tags for this repo, V1 should have been updated already?

**Comment by kyle-rader** [comment]
`set-env` is now disabled, and I am seeing Errors now using 
```yml
- name: Setup .NET Core
      uses: actions/setup-dotnet@v1
      with:
        dotnet-version: 3.1.302
```

**Comment by ericsampson** [comment]
This can probably be closed then? Phew :)

**Comment by vsafonkin** [comment]
Issue was fixed in #135 , closed.
